front/app/components/FormBuilder/edit/index.tsx

```diff
@@ -247,6 +283,13 @@ export const FormEdit = ({
                         />
                       </Box>
                     )}
+                    {editedAndCorrect && (
```

Comment:
This is the bit that will need combining with my permissions warning messages to make sure only one warning message appears at any one time